webapp/src/ts/components/duplicate-contacts/duplicate-contacts.component.ts

```diff
@@ -33,44 +33,30 @@ import { Contact } from '@medic/cht-datasource';
   ]
 })
 export class DuplicateContactsComponent {
-  @Input() duplicate: Contact.v1.Contact | undefined;
+  @Input() duplicate?: Contact.v1.Contact;
   isLoading: boolean = false;
-  error?: string = undefined;
-  _summary;
-
-  private readonly omitProperties = ['reported_date', 'name'];
+  error?: string;
+  contactSummary?: Record<string, unknown>;
 
   constructor(
     private readonly router: Router,
     private readonly formService: FormService,
     private readonly translateService: TranslateService
   ){}
 
-  _navigateToDuplicate() {
+  navigateToDuplicate() {
     this.router.navigate(['/contacts', this.duplicate!._id]);
   }
   
-  _loadContactSummary = async () => {
-    if (!this.duplicate?._id){
-      return;
-    }
-    
-    if (this.isLoading){
+  loadContactSummary = async () => {
+    if (this.contactSummary || !this.duplicate?._id || this.isLoading){
```

Comment:
I added `this.contactSummary` to the check here so we don't re-load the contact summary once it has been calculated (e.g. if the user just opens and closes the panel repeatedly).